lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -42,6 +42,10 @@ All templates should be in the C<templates> directories of the application,
 which can be customized with L<Mojolicious::Renderer/"paths">, or one of the
 the C<DATA> sections from L<Mojolicious::Renderer/"classes">.
 
+Any layouts should be directly under one of the C<templates> directories of the
+application (for example C<templates/layouts/example.layout.html.ep>) or one of
+the C<DATA> sections from L<Mojolicious::Rendered/"classes">.
```

Comment:
I was about to say the same thing. Also `example.layout.html.ep` is a bad name because it makes it look like `.layout.` is magical somehow. Later examples use `mylayout.html.ep`